src/components/v2/common/ReactSelect.utils.tsx

```diff
@@ -52,6 +52,7 @@ export const getCommonSelectStyle = (styleOverrides = {}) => {
         valueContainer: (base) => ({
             ...base,
             padding: '0 8px',
+            fontWeight: '400',
```

Comment:
Is it true for all react select?

Reply:
Yes. It should have a 400 font weight by default.